training/dist_clm_train.py

```diff
@@ -345,6 +345,8 @@ def main():
         
     if args.total_steps is None:
         args.total_steps = len(train_data_loader)
+
+    args.epoch_steps = int(len(train_data_loader) / args.batch_size)
```

Comment:
I don't think we should be setting properties on args that are not passed in via CLI. Can we compute this value in `train_loop`?

Reply:
k

Reply:
fixed